includes/class-press-sync.php

```diff
@@ -1329,6 +1314,50 @@ public function change_the_next_page( $next_page = 1 ) {
 		return $next_page;
 	}
 
+	/**
+	 * Get the synced object IDs from options or the remote site.
+	 *
+	 * This gets reset whenever a sync is started on the first page, see Dashboard::sync_wp_data_via_ajax.
+	 *
+	 * @since NEXT
+	 *
+	 * @param  string $objects_to_sync The post type of the things to sync.
+	 * @return array
+	 */
+	public function get_synced_object_ids( $objects_to_sync ) {
+		$option_name = "ps_synced_post_session_{$objects_to_sync}";
+		$last_sync   = get_option( $option_name );
+
+		if ( is_array( $last_sync ) && ! empty( $last_sync ) ) {
+			return $last_sync;
+		}
+
+		$url = $this->get_remote_url( '', 'progress', array(
+			'post_type' => $objects_to_sync,
+		) );
+
+		$remote_get_args = array(
+			'timeout' => 30,
+		);
+
+		$response      = wp_remote_get( $url, $remote_get_args );
+		$response_code = wp_remote_retrieve_response_code( $response );
```

Comment:
This is cool, I didn't know you could do this. I always inspected the response array (which would change). :+1: